pkg/controllermanager/controller/plant/plant_test.go

```diff
@@ -172,5 +176,23 @@ var _ = Describe("Plant", func() {
 				Status: corev1.NodeStatus{Conditions: []corev1.NodeCondition{{Type: corev1.NodeReady, Status: corev1.ConditionFalse}}},
 			}, BeNil(), Not(BeTrue())),
 		)
+		DescribeTable("checkClusterNodes - returns correct condition on failure",
+			func(caseMatcher types.GomegaMatcher) {
+				var (
+					conditionEveryNodeReady = helper.InitCondition(gardencorev1alpha1.PlantEveryNodeReady)
+					discoveryMockclient     = mockdiscovery.NewMockDiscoveryInterface(ctrl)
+					runtimeClient           = mockclient.NewMockClient(ctrl)
```

Comment:
See above.

Reply:
RuntimeClient not required by test checkAPIServerAvailablility so I wont include it in beforeEach